cluster/containers.go

```diff
@@ -0,0 +1,64 @@
+package cluster
+
+import (
+	"strings"
+
+	"github.com/docker/docker/pkg/stringid"
+)
+
+// Containers represents a list a containers
+type Containers []*Container
+
+// Match returns a container using it's ID or Name
+func (containers Containers) Match(IDOrName string) *Container {
```

Comment:
I'd use something else than `Match` since this doesn't match (a match is typically a boolean operation).

`Find` or `Get` might work